Add instructions for log patterns and parsing rules (#5976)
DataDog/documentation: content/en/logs/explorer/patterns.md

## See log trends and newcomers over time

Switch to the patterns view to automatically see your log patterns for the selected context. A [context][1] is composed of a time range and a search query. Each pattern comes with highlights, to get you straight to its characteristic features:

* A mini graph displays a rough timeline for the volume of its logs, to help you identify how that pattern differs from other patterns.
* Sections of logs that vary within the pattern are highlighted to help you quickly identify differences across log lines.

Click on a pattern to see a sample of underlying logs, and eventually display one single log.

{{< img src="logs/explorer/patterns/patterns_overview.mp4" alt="Log Patterns" video="true" responsive="true" width="90%" >}}

**Note**: The pattern detection is based on 10,000 log samples. Refine the search to see patterns limited to a specific subset of logs.

## Pattern Actions

{{< img src="logs/explorer/patterns/patterns_action.png" alt="Log Patterns Actions" responsive="true" style="width:80%;">}}

> When slecting a pattern, in addition of displaying the pattern and a sample of underlying logs, three buttons are available in the top-right corner of the panel to:

Reviewer: ```suggestion
Selecting a pattern opens a contextual panel that displays a sample of underlying logs. Three buttons are available in the top-right corner:
```
